analyzers/its/expected/BlazorSample/BlazorSample--net7.0-S108.json

```diff
@@ -0,0 +1,30 @@
+{
+"issues":  [
```

Comment:
Only the local function is detected => Rule improvement issue? 
No issues in razor files due to the limitation of the SL context => OK.

Reply:
Looking at the rule implementation, it fails the IsNestedAndEmpty method:
```cs
private static bool IsNestedAndEmpty(BlockSyntax node) =>
    IsNested(node) && node.IsEmpty();
```
We are explicitly checking that the parent kind is not one of the AllowedContainerKinds (SyntaxKind.ConstructorDeclaration, SyntaxKind.MethodDeclaration..) and we have test coverage in place to cover this behavior. I think this was introduced to limit the noise / FP rate?

Reply:
Oh OK, I see. Not sure if it's a noise/FP issue, but since the design takes a clear position on the matter, we can indeed leave it as it is.